deps/build.jl

```diff
@@ -1,16 +1,21 @@
 using Compat
 
+const condadir = abspath(homedir(), ".julia", "conda")
+const condadeps = joinpath(condadir, "deps.jl")
+
 module DefaultDeps
     using Compat
     if isfile("deps.jl")
         include("deps.jl")
-    end
-    if !isdefined(@__MODULE__, :ROOTENV)
-        const ROOTENV = abspath(dirname(@__FILE__), "usr")
+    elseif isfile(Main.condadeps)
+        include(Main.condadeps)
     end
     if !isdefined(@__MODULE__, :MINICONDA_VERSION)
         const MINICONDA_VERSION = "3"
     end
+    if !isdefined(@__MODULE__, :ROOTENV)
+        const ROOTENV = joinpath(Main.condadir, MINICONDA_VERSION)
```

Comment:
I suggested `joinpath(Main.condadir, "base")` and let `conda` control the Python version for each Julia environment https://github.com/JuliaPy/Conda.jl/issues/123#issuecomment-423376252.  But I guess it's OK for a quick fix.

Reply:
`conda` command installed in Miniconda3 can install any Python version registered in anaconda repo like 2.7.  So, my thought was that, if you need Python 2 for Julia environment `~/.julia/environments/v1.0-with-py27` then you'd create conda environment `~/.julia/conda/envs/v1.0-with-py27` using `~/.julia/conda/base/bin/conda` (which is interpreted by Python 3).  But since such mechanism does not exist (and not easily implementable without package options), I thought switching Python 2 and 3 by `ENV["CONDA_JL_VERSION"]` is a valid workaround.